### DISCUSSION TITLE
build the admin ui on build time

### DISCUSSION THREAD

**Discussion by macrozone** [vendurehq/vendure#1263] [Q&A] [answered]
the AdminUiPlugin compiles angular code on startup. This is not suitable for container hosting or cloud hosting in general as it adds cpu and memory load on every pod that starts (in my case it even goes out of memory).

 Is it possible to build it separatly during build time?

**Comment by michaelbromley** [comment, marked as answer]
Yes, see these docs: https://www.vendure.io/docs/plugins/extending-the-admin-ui/#compiling-as-a-deployment-step

**Reply by macrozone** [reply]
thank you for the fast response,
